ambry-server/src/main/java/com.github.ambry.server/AmbryRequests.java

```diff
@@ -921,33 +959,42 @@ private void sendGetResponse(RequestResponseChannel requestResponseChannel, GetR
    * Check that the provided partition is valid, on the disk, and can be written to.
    * @param partition the partition to validate.
    * @param requestType the {@link RequestOrResponseType} being validated.
+   * @param skipConditionsCheck whether to skip ({@code true}) or perform conditions check.
```

Comment:
variable needs to be named better to describe what checks are being skipped (and doc too)